src/wazuh_db/helpers/wdb_global_helpers.h

```diff
@@ -0,0 +1,336 @@
+/*
+ * Wazuh DB helper module for agents database
+ * Copyright (C) 2015-202, Wazuh Inc.
```

Comment:
There is a missing number : `(C) 2015-202`

Reply:
Fixed!